examples/gallery/embellishments/pygmt_logo.py

```diff
@@ -0,0 +1,362 @@
+    box=None,  # -> use box parameter of Figure.image
+):
+    """
+    Docstrings
+    """
+
+    # Start of subfunction
+
+    def create_logo(  # noqa: PLR0915
+        black_white=black_white,
+        dark_mode=dark_mode,
+        hex_shape=hex_shape,
+        wordmark=wordmark,
+        orientation=orientation,
+    ):
+        # -----------------------------------------------------------------------------
+        # Helpful definitions
+        # -----------------------------------------------------------------------------
+        size = 4
+        region = [-size, size] * 2
+
+        xy_yellow_1 = 2.65
+        xy_yellow_2 = 1.3
+
+        no_line = "cyan@100"
+        no_fill = f"+g{no_line}"
+
+        # Rotation around z (vertical) axis placed in the center
+        # Has to be applied to each plotting command, up on second call set to True
+        # Do NOT rotated initial basemap
+        angle_rot = 30  # degrees
+        perspective = f"{angle_rot}+w0/0"
+
+        # -----------------------------------------------------------------------------
+        # Define colors
+        # -----------------------------------------------------------------------------
+        color_dark = "gray20"
+        color_light = "white"
+
+        # visual
+        color_blue = "48/105/152"  # Python blue
+        color_yellow = "255/212/59"  # Python yellow
+        color_red = "238/86/52"  # GMT red
+        if black_white:
+            color_blue = color_yellow = color_red = color_light
+            if not dark_mode:
+                color_blue = color_yellow = color_red = color_dark
+
+        # background and wordmark
+        color_bg = color_dark
+        color_py = color_yellow
+        color_gmt = color_light
+        if not dark_mode:
+            color_bg = color_light
+            color_py = color_blue
+            color_gmt = color_dark
+
+        # -----------------------------------------------------------------------------
+        # Define shape
+        # -----------------------------------------------------------------------------
+        symbol = "c"  # circle
+        diameter = 7.5
+        diameter_add = 0.5
+        if hex_shape:
+            symbol = "h"  # hexagon
+            diameter = 8.6
+            diameter_add = 0.6
+
+        # -----------------------------------------------------------------------------
+        # Define wordmark
+        # -----------------------------------------------------------------------------
+        match orientation:
+            case "vertical":
+                proj_wm = f"X{size * 2 - 1.5}c/{size * 2}c"
+                pos_wm = f"jTC+o0c/0.2c+w{size * 2 - 2.7}c"
+                args_text_wm = {"x": -3.2, "y": -2.8, "justify": "LM"}
+            case "horizontal":
+                proj_wm = f"X{size * 2}c/{size - 2}c"
+                pos_wm = f"jLM+o0.1c/0c+w{size - 2}c"
+                args_text_wm = {"x": -1.5, "y": 0, "justify": "LM"}
```

Comment:
See my comment below. The "x"/"y" are the coordinates relative to the current basemap (with `region=[-4, 4, -4, 4]`). The x/y coordinates and the font size may not be the most suitable values.
```suggestion
        match orientation:
            case "vertical":
                args_text_wm = {"x": 0, "y": -4.5, "justify": "CT", "font": "2.5c,AvantGarde-Book"}
            case "horizontal":
                args_text_wm = {"x": 4.5, "y": 0, "justify": "LM", "font": "8c,AvantGarde-Book"}

```

Reply:
Interesting that it is possible to adjust the font size based on centimeters! Did not know this.